src/libraries/System.Runtime/tests/TrimmingTests/DefaultValueAttributeCtorTest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reflection;
+
+/// <summary>
+/// Tests that System.ComponentModel.TypeConverter.ConvertFromInvariantString
+/// is not trimmed out when needed by DefaultValueAttribute in a trimmed application.
+/// </summary>
+class Program
+{
+    static int Main(string[] args)
+    {
+        var attribute = new DefaultValueAttribute(typeof(string), "Hello, world!");
+
+        // There's a fallback in the DefaultValueAttribute ctor for when the following
+        // value is null. It shouldn't be null at this point, if the linker didn't trim
+        // System.ComponentModel.TypeConverter.ConvertFromInvariantString out.
+        object convertFromInvariantString = typeof(DefaultValueAttribute).GetField(
+            "s_convertFromInvariantString",
+            BindingFlags.GetField | BindingFlags.NonPublic | BindingFlags.Static).GetValue(attribute);
```

Comment:
The field is static, so you don't need to pass `attribute` into `GetValue`. Instead you just pass `null` for statics.

Reply:
I've removed this reflection code since we wouldn't get the expected value if the func were null.